frame/staking/src/migrations.rs

```diff
@@ -20,6 +20,36 @@ use super::*;
 use frame_election_provider_support::SortedListProvider;
 use frame_support::traits::OnRuntimeUpgrade;
 
+pub mod v11 {
+	use super::*;
+	use frame_support::{pallet_prelude::ValueQuery, storage_alias};
+
+	#[storage_alias]
+	type HistoryDepth<T: Config> = StorageValue<Pallet<T>, u32, ValueQuery>;
+
+	/// Clean up `HistoryDepth` from storage
+	///
+	/// We will be depending on the configurable value of `HistoryDepth` post
+	/// this release.
+	pub struct MigrateToV11<T>(sp_std::marker::PhantomData<T>);
+	impl<T: Config> OnRuntimeUpgrade for MigrateToV11<T> {
+		fn on_runtime_upgrade() -> frame_support::weights::Weight {
+			if StorageVersion::<T>::get() == Releases::V10_0_0 {
+				// may be just check its greater?
+				debug_assert_eq!(T::HistoryDepth::get(), HistoryDepth::<T>::get());
+				HistoryDepth::<T>::kill();
+				StorageVersion::<T>::put(Releases::V11_0_0);
+
+				log!(info, "MigrateToV11 executed successfully");
+				T::DbWeight::get().reads_writes(1, 1)
```

Comment:
```suggestion
				T::DbWeight::get().reads_writes(2, 2)
```

Reply:
I assume this is because there are 2 reads and 2 writes in the migration? Seems to me though some other migrations do not have the correct numbers that I would expect. Such as in v10, I think there is atleast 2 reads and 2 writes, in addition to call to `apply_slash`. Is that a mistake?